paper-radio-button.html

```diff
@@ -71,7 +71,7 @@
       hostAttributes: {
         role: 'radio',
         'aria-checked': false,
-        tabindex: 0
+        tabindex: -1
```

Comment:
More specifically I think this is only an issue if you create paper-radio-button with no group or selected button.

I will look at leaving tabindex: 0 in the hostAttributes, use the attached callback set to -1 if the button is in a group.  Same condition will be applied in _checkChanged so tabindex will always be 0 if outside a paper-radio-group.

Now to figure out how to detect paper-radio-group in parent DOM.  Should tabindex be dynamically managed if paper-radio-group any parent node, or only the direct parent?  The plan is to check the parent structure for *[role="radiogroup"] instead of searching for paper-radio-group.  This seems to be the right way to do it, let me know if you disagree.